### PR TITLE
Moved language switcher to user settings

### PR DESCRIPTION
**Pull request by melegiul**
### Short description

Moved language switcher to user settings

### Proposed changes

- Removed language switcher from top menu bar.
- Added language switcher to user settings.

### Resolved issues

Fixes: #651

### CODE REVIEW

File: src/cms/templates/settings/user.html [hunk 1]
```diff
@@ -90,6 +88,28 @@ <h3 class="heading font-bold text-white">{% trans 'Two-Factor-Authentication' %}
...
+                <label for="language" class="font-bold p-2 mb-1 cursor-pointer mr-32">{% trans 'Choose language:' %}</label>
+                <select name="language">
+                    {% get_current_language as LANGUAGE_CODE %}
+                    {% get_available_languages as LANGUAGES %}
+                    {% for language_code, language_name in LANGUAGES %}
+                        <option value="{{ language_code }}"{% if language_code == LANGUAGE_CODE %} selected{% endif %}>
+                            {{ language_code|language_name_local }}
+                        </option>
+                    {% endfor %}
+                </select>
```
Comment (timoludwig) [inline src/cms/templates/settings/user.html:107]:
```suggestion
                <div class="mb-2 pl-2">
                    {% trans "This setting only affects this device." %}
                </div>
                <div class="relative">
                    <select name="language" class="block appearance-none w-full bg-gray-200 border border-gray-200 text-gray-800 py-3 px-4 pr-8 rounded leading-tight focus:outline-none focus:bg-white focus:border-gray-400">
                        {% get_current_language as LANGUAGE_CODE %}
                        {% get_available_languages as LANGUAGES %}
                        {% for language_code, language_name in LANGUAGES %}
                            <option value="{{ language_code }}"{% if language_code == LANGUAGE_CODE %} selected{% endif %}>
                                {{ language_code|language_name_local }}
                            </option>
                        {% endfor %}
                    </select>
                    <div class="pointer-events-none absolute inset-y-0 right-0 flex items-center px-2 text-gray-800">
                        <img src="{% static 'svg/select-down-arrow.svg' %}" class="fill-current h-4 w-4" />
                    </div>
                </div>
```

**Review by timoludwig — COMMENTED** [review]
Looks good! :+1:

I'd just add a small help text that the setting only affects the current device and adjust the classes of the select tag so it matches our other dropdown lists.

File: src/cms/templates/settings/user.html (hunk 1 above)
Comment (melegiul) [reply src/cms/templates/settings/user.html]: Yes, the select box looks better now, I just updated the translation file, now it should work.

**Review by timoludwig — APPROVED** [review]
Perfect :tada:

**Pull request merged**